utils/src/test/kotlin/RedirectionTest.kt

```diff
@@ -33,7 +33,7 @@ class RedirectionTest : WordSpec({
 
         "work for stdout only" {
             val stdout = redirectStdout {
-                for (i in 1..numberOfLines) System.out.println("stdout: $i")
+                for (i in 1..numberOfLines) println("stdout: $i")
```

Comment:
I came across this inspection hint recently, too, but decided to keep the explicit `System.out.` here to mirror the required `System.err.` below.

Reply:
I removed the commit.